src/style.css

```diff
@@ -1 +1,47 @@
-/* add styles here */
+body {
+  margin: 0;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  background-image: url(images/star.svg);
+  background-repeat: no-repeat;
+  background-position: center;
+  height: 16px;
+  width: 16px;
+}
+
+.stars__star:not(:first-of-type, :last-of-type) {
+  margin: 0 2px;
+}
+
+.stars__star:first-of-type {
+  margin-right: 2px;
+}
+
+.stars__star:last-of-type {
+  margin-left: 2px;
+}
```

Comment:
Looks weird, just use margin-right: 4px and remove margin-right from the last stars in a row using :last-child rule